modules/planning/common/frame_open_space.h

```diff
@@ -101,6 +114,14 @@ class FrameOpenSpace {
 
   void AddObstacle(const Obstacle &obstacle);
 
+  bool VPresentationObstacle();
+
+  common::Status ObsHRep(const std::size_t &nOb, const Eigen::MatrixXd &vOb,
+                         const std::vector<std::vector<Vec2d>> &lOb,
+                         Eigen::MatrixXd *A_all, Eigen::MatrixXd *b_all);
+
+  bool HPresentationObstacle();
```

Comment:
H means horizontal? Please try to avoid using abbreviations, especially for public functions.